docs/pt_BR/source/conf.py

```diff
@@ -41,49 +48,49 @@
 #
 # source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
-
+source_encoding = 'utf-8-sig'
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
-project = u'correios'
-copyright = u'2017, olist'
-author = u'olist'
+project = 'correios'
+copyright = '2018, olist'
+author = 'olist'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'3.1.2'
+version = '3.1.2'
```

Comment:
this is the short version, so here should be `'3.1'`. to be more accurate correios now is on version `4.7.0` so `4.7` is the correct value here